actions/views/create_comment.tsx

```diff
@@ -46,12 +46,16 @@ export function clearCommentDraftUploads() {
 // we're on will not save the draft quickly enough on page unload.
 export function updateCommentDraft(rootId: string, draft?: PostDraft | null) {
     const key = `${StoragePrefixes.COMMENT_DRAFT}${rootId}`;
+    let updatedDraft = draft;
     if (draft) {
-        localStorage.setItem(key, JSON.stringify(draft));
+        const item = localStorage.getItem(key);
+        const data = item ? JSON.parse(item) : {};
+        updatedDraft = {...draft, createAt: data.createAt || new Date()};
+        localStorage.setItem(key, JSON.stringify(updatedDraft));
     } else {
         localStorage.removeItem(key);
     }
-    return setGlobalItem(key, draft);
+    return setGlobalItem(key, updatedDraft);
```

Comment:
I noticed that recently too. I'm currently working on the drafts code, and I might end up cleaning that up myself once I untangle all of that, so we can leave this as-is for now. Web Platform has plans to redo all of the storage-related code in Q4 anyway.

Reply:
AFAICT setting it to `null` had no actual effect. It seems that 
`redux-persist` will ignore it and wouldn't update the item in the storage.
So I had to set it to an empty string to remove the draft.

It works right now I think because we are always setting the message to something,
either an empty string or an actual message.

That said, I would like to keep drastic changes in this PR at minimun, since it already has a ton of additions.